docs/third-party-developers/extensibility/data-store/checkout.md

```diff
@@ -292,6 +293,22 @@ const store = select( CHECKOUT_STORE_KEY );
 const isCalculating = store.isCalculating();
 ```
 
+
+### prefersCollection
+
+Returns true if the customer prefers to collect their order, and false otherwise.
+
+#### _Returns_ <!-- omit in toc -->
+
+-   _prefersCollection_ `boolean`: True if the shopper prefers to collect their order.
+
+#### _Example_ <!-- omit in toc -->
+
+```js
+const store = select( 'wc/store/checkout' );
```

Comment:
In https://github.com/woocommerce/woocommerce-blocks/pull/11986, we were using
```js
const store = select( CHECKOUT_STORE_KEY );
```
instead of
```js
const store = select( 'wc/store/checkout' );
```

For consistency-reasons, I suggest using one or the other and ensure that all docs are using the same format.